### PR TITLE
napari installation

### PR DESCRIPTION
**Pull request by tischi**
I am working on updating the napari installation.

See also here: https://forum.image.sc/t/versioned-conda-installations-for-napari/77942/16

Questions: 
1. for what do we need matplotlib?
2. do we really need this`napari-brightness-contrast?
3. do we really need this `napari-plot-profile`?

I would like to keep our installation as minimal as possible even if that means that one cannot do certain things in some of the teaching modules.

### CODE REVIEW

**Comment by tischi** [comment]
I am a bit torn...

I think it would be nicer to do all visualisation in napari, which would speak for the plugins, but it is also nice to only depend on rock-solid stuff, which speaks for matplotlib.

One line of arguing could be:

If you want to visualise images => use napari, it is made for it
If you want to visualise any kind of plots (line profile or histogram) napari is not made for it, matplotlib is made it => use matplotlib.

=> remove the plugins and do the plots with matplotlib.

What do you think?

**Comment by k-dominik** [comment]
I would propose to do this with an environment file. Doing the installation in multiple steps has 0 benefits. Pip dependencies can also be incorporated, but I'd suggest to really make sure that pip doesn't install/change anything that is already in the environment (so add _all_ necessary dependencies, also of the pip packages, with compatible versions to the mamba/conda part of the env file).

**Comment by tischi** [comment]
> Doing the installation in multiple steps has 0 benefits.

I was hoping that maybe it could now in done in _one_ step?! Just one line of mamba?

File: _includes/tool_installation/activities/tool_installation_napari.md [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
+### Installation (only once)
 
-#### Installation (only once)
-
-1. install miniconda (TODO)
-1. `conda create -n skimage-napari-tutorial python=3.9`
-1. `conda activate skimage-napari-tutorial`
+1. [install mamba](https://github.com/conda-forge/miniforge#mambaforge)
+1. open a terminal window
+1. `mamba create -c conda-forge -n napari-0.4.17 python=3.10 napari=0.4.17 notebook`
```
Comment (tischi) [inline _includes/tool_installation/activities/tool_installation_napari.md:7]: name of env: `skimage-napari-tutorial`

**Pull request merged**